server/config/passport.js

```diff
@@ -141,10 +152,9 @@ passport.use(new GoogleStrategy({
 
     User.findByEmail(primaryEmail, (findByEmailErr, existingEmailUser) => {
       let username = profile._json.emails[0].value.split('@')[0];
-      User.findByUsername(username, (findByUsernameErr, existingUsernameUser) => {
+      User.findByUsername(username, true, (findByUsernameErr, existingUsernameUser) => {
```

Comment:
What does `true` do here? A comment would be useful I think.

Reply:
Good call 😄 I'll update this!